modules/attributes-openshift-dedicated.adoc

```diff
@@ -5,8 +5,10 @@
 :experimental:
 :imagesdir: images
 :OCP: OpenShift Container Platform
-:OCM: OpenShift Cluster Manager (OCM)
-:console-redhat-com: link:https://console.redhat.com/openshift[OpenShift Cluster Manager (OCM)]
+:cluster-manager-first: Red Hat OpenShift Cluster Manager
+:cluster-manager: OpenShift Cluster Manager
+:cluster-manager-url: link:https://console.redhat.com/openshift[OpenShift Cluster Manager]
+:cluster-manager-url-pull: link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site]
```

Comment:
Change "pull secret from the Red Hat OpenShift Cluster Manager site" to "pull secret from Red Hat OpenShift Cluster Manager"